--- a/kfs-core/src/main/java/org/kuali/kfs/kns/util/WebUtils.java
+++ b/kfs-core/src/main/java/org/kuali/kfs/kns/util/WebUtils.java
@@ -749,7 +749,8 @@ public static String getPrincipalDisplayName(final String principalId) {
         if (person == null) {
             return "";
         } else {
-            return person.getName();
+            // CU Customization: Return potentially masked Person name instead.
+            return person.getNameMaskedIfNecessary();
         }
     }
 
