--- a/kfs-core/src/main/java/org/kuali/kfs/kew/routeheader/DocumentRouteHeaderValueActionListExtension.java
+++ b/kfs-core/src/main/java/org/kuali/kfs/kew/routeheader/DocumentRouteHeaderValueActionListExtension.java
@@ -53,7 +53,8 @@ private void initializeInitiatorName() {
         if (!isInitiatorNameInitialized) {
             final Person person = SpringContext.getBean(PersonService.class).getPerson(getInitiatorPrincipalId());
             if (person != null) {
-                initiatorName = person.getName();
+                // CU Customization: Return potentially masked Person name instead.
+                initiatorName = person.getNameMaskedIfNecessary();
             }
             isInitiatorNameInitialized = true;
         }
