--- a/kfs-core/src/main/java/org/kuali/kfs/kew/impl/document/search/DocumentSearchCriteriaBo.java
+++ b/kfs-core/src/main/java/org/kuali/kfs/kew/impl/document/search/DocumentSearchCriteriaBo.java
@@ -265,7 +265,8 @@ public Person getInitiatorPerson() {
     public String getInitiatorDisplayName() {
         if (StringUtils.isNotBlank(initiatorPrincipalId)) {
             final Person person = SpringContext.getBean(PersonService.class).getPerson(getInitiatorPrincipalId());
-            return person == null ? initiatorPrincipalId : person.getName();
+            // CU Customization: Return potentially masked Person name instead.
+            return person == null ? initiatorPrincipalId : person.getNameMaskedIfNecessary();
         }
 
         return initiatorPrincipalId;
